File: src/main/resources/static/js/registroasistencia.js
/* 
 * Click nbfs://nbhost/SystemFileSystem/Templates/Licenses/license-default.txt to change this license
 * Click nbfs://nbhost/SystemFileSystem/Templates/Other/javascript.js to edit this template
 */
$(document).ready(function () {
    listar();
});
function listar() {
    $.ajax({url: "/RegistroAsist/all",
        type: 'GET',
        success: function (x) {
            $("#table tbody tr").remove();
            x.forEach((item, index, array) => {
                $("#table").append(
                        "<tr>\n\
                            <td>" + (index + 1) + "</td>\n\
                            <td>" + item.nombre + "</td>\n\
\n\                        <td>" + item.fecha + "</td>\n\
\n\                        <td>" + item.hora + "</td>\n\
\n                      <td>" + item.descripcion + "</td>\n\
                         <td>" + item.lugar + "</td>\n\
\n\                      <td>" + item.direccion + "</td>\n\
\n\                          <td>" + item.programa + "</td>\n\
                            <td style='text-align: center'>\n\
                                <a href='#' onclick='editar(" + item.regisId + ")'><i class='fa-solid fa-pen-to-square yelow'></i></a>\n\
                            </td>\n\
                            <td style='text-align: center'>\n\
                                <a href='#' onclick='eliminar(" + item.regisId + ")'><i class='fa-solid fa-trash-can red'></i></a>\n\
                            </td>\n\
                        </tr>");

            });
        }
    });
}

File: src/main/java/upeu/ayllusinchi/entity/RegistroAsistencia.java
/*
 * Click nbfs://nbhost/SystemFileSystem/Templates/Licenses/license-default.txt to change this license
 * Click nbfs://nbhost/SystemFileSystem/Templates/Classes/Class.java to edit this template
 */
package upeu.ayllusinchi.entity;

import java.io.Serializable;
import javax.persistence.Column;
import javax.persistence.Entity;
import javax.persistence.GeneratedValue;
import javax.persistence.GenerationType;
import javax.persistence.Id;
import javax.persistence.Table;
import lombok.Data;

/**
 *
 * @author User
 */
@Data
@Entity
@Table(name ="RegistroAsis")
public class RegistroAsistencia implements Serializable {
     @Id
@Column(name = "Regis_id")
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    private Long regisId;
 @Column(name = "Nombre")
    private String nombre;
    @Column(name = "Fecha")
    private String fecha;
 @Column(name = "Hora")
    private String hora;
    @Column(name = "Descripcion")
    private String descripcion;
 @Column(name = "Lugar")
    private String Lugar;
    @Column(name = "Direccion")
    private String direccion;
 @Column(name = "Programa")
    private String programa;
   

    
}
